### ISSUE TITLE
Parse Gemini 2.3 metadata 

### ISSUE THREAD

**Issue by EFT-Defra** [Defra-Data-Science-Centre-of-Excellence/sds-data-model#1]
Spatial datasets published by the UK government should have associated metadata that conforms to the [UK GEMIN standard](https://www.agi.org.uk/uk-gemini/). Ultimately, we want to parse that metadata into a python dictionary that can be attached to a [xarray DataArray or Dataset](https://xarray.pydata.org/en/stable/user-guide/data-structures.html) via [the `attrs` property ](https://xarray.pydata.org/en/stable/getting-started-guide/faq.html?highlight=attrs#what-is-your-approach-to-metadata).

[Summary of metadata elements](https://www.agi.org.uk/gemini/40-gemini/1250-element-summary/)

[Dataset of series encoding guidelines](https://www.agi.org.uk/gemini/40-gemini/1062-gemini-datasets-and-data-series/)

I think we should break this down into the follow tasks:

- [x] #34
- [x] #35
- [ ] #36
- [ ] #37
- [ ] #38
- [ ] #39

**Comment by s-flint-defra** [comment]
@EFT-Defra I notice there hasnt been much activity on GEMINI metadata recently. Is this due t lack of time/resource and so should stay in the backlog?

**Comment by EFT-Defra** [comment]
@s-flint-defra, you're correct, there hasn't been any work on this for a while and that's been due to resource constraints. In fact, I don't think there's been any work on this since Jordan left. He summarised where he'd go to here: #54.

I still think this work is important, because valid Gemini metadata files provide a lot of information that might be useful to analysts (like provenance and access / licence restrictions), but there's quite a lot to do and it isn't essential to an MVP.

As well as parsing the individual metadata files, we'll also need to think about how we'd could combine metadata when combining datasets.
